### ISSUE TITLE
Exchanging PDO/SDO data 

### ISSUE THREAD

**Issue by bbhk17** [CANopenNode/CANopenNode#93]
Hi Guys, 
First great work ! 

I did looked at the libedssharp to build my eds file and the code.
I'm on STM32, so I have to do my driver. No problem so far.

My question is more about **"how I give my data to the TPDO ?"** Or how I get my data back from RPDO ?

I used the DS301 profile, my PDO mapping will be static, but I still have the 0x1800 / 0x1A00 SDO.
I went thought the whole CO_PDO.c, but I can't find my way. 

What's this SDO (0x1A00 PDM Mapping) how it is working ? Do you have some good documentation I could use ?

Thx !

**Comment by bbhk17** [comment]
Hi Robin,

Thanks for the answer, one thing that was missing is that the TPDO and RPDO are SDO ( I put them at address 0x5000+ on manufacturer object) 
then I linked them on the TPDO/RPDO tab, 
this link made the mapping parameter to 0x50000108. which means the SDO object that I configure before.
Then this is correct, I've got a "CO_OD_RAM.txPDO1[0]" 

I assume (not tested) that this is the variable ( i called it TxPDO1 on the eds config soft) I need to write, for sending.

I need to finish to port the driver, then I'll close the "issue"

Thanks again
